### PR TITLE
Website build pack: a lot of different improvements

### PR DESCRIPTION
**Pull request by Y-Shcheholskyy**
- Created getting started layout;
- Created Example page;
- Created educational layout (for examples, guide);
- Added jquery lib;
- Added jekyll-toc plugin;
- Added benchmark graph on the main page.

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -8,7 +8,6 @@
     {%- include header.html -%}
 
     <main class="page-content" aria-label="Content">
-      <!-- <div class="wrapper"> -->
         {{ content }}
       </div>
```
Comment (bhoran) [inline _layouts/base.html:12]: Shouldn't this also be commented out?

**Comment by bhoran** [comment]
Why have you changed some permalinks to use underscore instead of hyphen, but not all of them?

File: _layouts/base.html (hunk 1 above)
Comment (Y-Shcheholskyy) [reply _layouts/base.html:12]: I'm sorry, I don't understand, wrapper not needed here.

Comment (bhoran) [reply _layouts/base.html:12]: Sorry, the comment isn't clear about which line I mean. I think you need to comment out the closing <div> on line 13/12

Comment (Y-Shcheholskyy) [reply _layouts/base.html]: got it, just to remove "</div>", done :)

**Pull request merged**